### PR TITLE
[dev-v2.7] Fix patch failures for newer versions of charts-build-scripts

### PR DESCRIPTION
**Pull request by nicholasSUSE**
## Problem
We have a series of charts that should have been updated to not break the CI. 
Their packages folder needed to adapt to the new versions of `charts-build-scripts`

## Solution
Provide the necessary adaptations for the charts in the packages folder when possible. 

Release a new patch update when just updating the packages is not possible.

### CODE REVIEW

**Review by w13915984028 — APPROVED** [review]
`Harvester-cloud-provider` looks good, thanks.

File: packages/harvester/harvester-cloud-provider/generated-changes/patch/Chart.yaml.patch [hunk 1]
```diff
@@ -21,3 +21,7 @@
  dependencies:
  - name: kube-vip
    repository: file://./charts/kube-vip
+-  version: 0.4.2
```
Comment (lucasmlp) [inline packages/harvester/harvester-cloud-provider/generated-changes/patch/Chart.yaml.patch:24]: This should be the only change in this file

File: packages/rancher-monitoring/rancher-project-monitoring/generated-changes/patch/Chart.yaml.patch [hunk 2]
```diff
@@ -10,3 +10,11 @@
    catalog.cattle.io/release-name: rancher-project-monitoring
  apiVersion: v2
  appVersion: 0.59.1
+@@ -10,7 +12,6 @@
+ - condition: grafana.enabled
+   name: grafana
+   repository: file://./charts/grafana
+-  version: 6.38.6
```
Comment (lucasmlp) [inline packages/rancher-monitoring/rancher-project-monitoring/generated-changes/patch/Chart.yaml.patch:17]: This should be the only change in this file

File: packages/epinio/epinio/generated-changes/patch/Chart.yaml.patch [hunk 3]
```diff
@@ -18,7 +18,29 @@
  apiVersion: v2
  appVersion: v1.9.0
  dependencies:
-@@ -27,7 +38,7 @@
+@@ -9,29 +20,25 @@
+   repository: file://./charts/dex
+   tags:
+   - dex
+-  version: 0.14.3
```
Comment (lucasmlp) [inline packages/epinio/epinio/generated-changes/patch/Chart.yaml.patch:25]: The dependencies version should be the only changes in this file

**Review by lucasmlp — CHANGES_REQUESTED** [review]

**Review by joshmeranda — APPROVED** [review]

**Review by lucasmlp — CHANGES_REQUESTED** [review]
LGTM 🚀

**Review by lucasmlp — APPROVED** [review]
LGTM 🚀

**Review by thardeck — APPROVED** [review]

**Review by mgfritch — APPROVED** [review]
`sriov` chart looks good, thanks!

File: release.yaml [hunk 4]
```diff
@@ -1,8 +1,18 @@
+rancher-istio:
```
Comment (nicholasSUSE) [inline release.yaml:1]: These versions of rancher-istio and fleet should not be released on the future.
They are just for fixing the patching issues.

**Review by andreas-kupries — COMMENTED** [review]
lgtm

**Pull request merged**